doc/api/tls.md

```diff
@@ -789,8 +790,6 @@ added: v0.11.3
 * `port` {number}
 * `host` {string}
 * `options` {Object}
-  * `host` {string} Host the client should connect to.
```

Comment:
Yep, that can be a separate PR. I'd suggest just leaving the host and port fields in the options description for now.